Dockerfile

```diff
@@ -1,8 +1,19 @@
-FROM ubuntu:latest
-
-EXPOSE 8555
-EXPOSE 8444
+FROM python:slim AS builder
+ARG BRANCH="main"
+RUN DEBIAN_FRONTEND=noninteractive apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y git build-essential
+RUN python -m venv /chia-blockchain/venv
+ENV PATH /chia-blockchain/venv/bin:$PATH
+WORKDIR /chia-blockchain/src
+RUN git clone --branch $BRANCH https://github.com/Chia-Network/chia-blockchain.git .
+RUN git submodule update --init mozilla-ca
+RUN pip install wheel
+RUN pip install --extra-index-url https://pypi.chia.net/simple miniupnpc==2.1
+RUN pip install --extra-index-url https://pypi.chia.net/simple .
 
+FROM python:slim AS starter
+LABEL maintainer="Chia Network Community"
```

Comment:
For now, I think we can skip labelling the image - might add a bunch of standard labels for our images later, but need to figure all that out

Reply:
So, what can I do now?